features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Operations/InlineEditableField.jsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update license header

Reply:
This file is don't required. will remove

Reply:
This file is don't required. will remove